packages/amplify-provider-awscloudformation/lib/configuration-manager.js

```diff
@@ -485,19 +490,24 @@ function removeProjectConfig(context) {
   }
 }
 
-async function loadConfiguration(context, awsClient) {
-  const projectConfigInfo = getCurrentConfig(context);
+async function loadConfiguration(context) {
+  const { envName } = context.amplify.getEnvInfo();
+  const config = await loadConfigurationForEnv(context, envName);
+  return config;
+}
+
+async function loadConfigurationForEnv(context, env) {
+  const projectConfigInfo = getConfigForEnv(context, env);
   if (projectConfigInfo.configLevel === 'project') {
     const { config } = projectConfigInfo;
+    let awsConfig;
     if (config.useProfile) {
-      const awsConfig = await systemConfigManager.getProfiledAwsConfig(context, config.profileName);
-      awsClient.config.update(awsConfig);
+      awsConfig = await systemConfigManager.getProfiledAwsConfig(context, config.profileName);
     } else {
-      awsClient.config.loadFromPath(config.awsConfigFilePath);
+      awsConfig = systemConfigManager.loadConfigFromPath(config.awsConfigFilePath);
```

Comment:
change this line to awsConfig =  context.amplify.readJsonFile(config.awsConfigFilePath);

Reply:
we do some additional check in there. I would prefer to keep this